pages/api/v1/dates/confirm-by.ts

```diff
@@ -0,0 +1,39 @@
+import { NextApiHandler, NextApiRequest, NextApiResponse } from 'next';
```

Comment:
Not finished copying over from registration-open.ts; any thoughts on how to refactor? I was trying to figure out how to pass an additional variable, like "date type" but the NextApiHandler only accepts req, res as parameters right?